modules/core/shared/src/main/scala/scaladex/core/model/GithubInfo.scala

```diff
@@ -61,7 +63,8 @@ case class GithubInfo(
       forks = forks,
       contributorCount = contributorCount,
       scalaPercentage = scalaPercentage,
-      license = license
+      license = license,
+      commitsPerYear = Option(commitActivity.map(_.total).sum)
```

Comment:
It would be better to have `None` instead of `Some(0)` when the `commitActivity` is not known.

```suggestion
      if (commitActivity.isEmpty) None else Some(commitActivity.map(_.total).sum)
```